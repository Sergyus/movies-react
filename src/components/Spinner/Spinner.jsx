import React from 'react';
import styles from './Spinner.scss';

const Spinner = () => (
  <div className={styles.spinner}>
    <div className={styles.ellipsis}>
      <div />
      <div />
      <div />
      <div />
    </div>
    <p className={styles.title}>Loading</p>
  </div>
);

export default Spinner;
