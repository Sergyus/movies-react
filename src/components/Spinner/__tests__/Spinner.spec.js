import React from 'react';
import { create } from 'react-test-renderer';
import Spinner from '..';

describe('Spinner component renders correctly', () => {
  it('Spinner renders correctly', () => {
    const tree = create(<Spinner />).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
