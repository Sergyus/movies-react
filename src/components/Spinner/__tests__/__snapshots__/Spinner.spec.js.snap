// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Spinner component renders correctly Spinner renders correctly 1`] = `
<div
  className="spinner"
>
  <div
    className="ellipsis"
  >
    <div />
    <div />
    <div />
    <div />
  </div>
  <p
    className="title"
  >
    Loading
  </p>
</div>
`;
